go/obscuronode/enclave/enclave.go

```diff
@@ -715,14 +721,15 @@ func DecryptWithPrivateKey(ciphertext []byte, priv *rsa.PrivateKey) ([]byte, err
 func getDBConnector(cfg config.EnclaveConfig) db.EthDBConnector {
 	if cfg.UseInMemoryDB {
```

Comment:
something to keep in mind for the next PR. Once the third condition is added, would be good to replace the ifs with a switch

Reply:
Yeah will definitely need a rethink tbh because the edgeless db stuff is going to require more config